cognite/neat/_rules/transformers/_converters.py

```diff
@@ -220,6 +224,140 @@ def _transform(self, rules: VerifiedRules) -> VerifiedRules:
         raise ValueError(f"Unsupported conversion from {type(rules)} to {self._out_cls}")
 
 
+_T_Entity = TypeVar("_T_Entity", bound=ClassEntity | ViewEntity)
+
+
+class ToExtension(RulesTransformer[DMSRules, DMSRules]):
+    def __init__(
+        self, new_model_id: DataModelIdentifier, prefix: str | None = None, mode: Literal["composition"] = "composition"
+    ):
+        self.new_model_id = DataModelId.load(new_model_id)
+        self.prefix = prefix
+        self.mode = mode
+
+    def transform(self, rules: DMSRules | OutRules[DMSRules]) -> JustRules[DMSRules]:
+        # Copy to ensure immutability
+        verified = self._to_rules(rules)
+        if self.prefix is None and verified.metadata.as_data_model_id() in COGNITE_MODELS:
```

Comment:
comment on the use case this covers would be helpful, as I can see below you allow for self.prefix to be None, but it is required if data model is actually Cognite_models. Why would it be, if you have option to strip `Cognite` from suffix if prefix is not provided. (maybe better name here is *affix* to make dist. from usage of prefix in Neat?)